tests/baselines/reference/parser.numericSeparators.binaryNegative.errors.txt

```diff
@@ -0,0 +1,62 @@
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/1.ts(1,5): error TS6188: Numeric seperators are not allowed at the end of a literal.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/2.ts(1,3): error TS1177: Binary digit expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/2.ts(1,3): error TS2304: Cannot find name '_110'.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/3.ts(1,2): error TS6188: Numeric seperators are not allowed at the end of a literal.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/3.ts(1,3): error TS1005: ';' expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/3.ts(1,3): error TS2304: Cannot find name 'B0101'.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/4.ts(1,5): error TS6188: Numeric seperators are not allowed at the end of a literal.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/4.ts(1,6): error TS1005: ';' expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/4.ts(1,6): error TS2304: Cannot find name '_11'.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/5.ts(1,12): error TS6188: Numeric seperators are not allowed at the end of a literal.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/5.ts(1,13): error TS1005: ';' expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/5.ts(1,13): error TS2304: Cannot find name '_'.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/6.ts(1,3): error TS1177: Binary digit expected.
+tests/cases/conformance/parser/ecmascriptnext/numericSeparators/6.ts(1,3): error TS2304: Cannot find name '___0111010_0101_1'.
+
+
+==== tests/cases/conformance/parser/ecmascriptnext/numericSeparators/1.ts (1 errors) ====
+    0b00_
+        ~
+!!! error TS6188: Numeric seperators are not allowed at the end of a literal.
+    
+==== tests/cases/conformance/parser/ecmascriptnext/numericSeparators/2.ts (2 errors) ====
+    0b_110
+      
+!!! error TS1177: Binary digit expected.
+      ~~~~
+!!! error TS2304: Cannot find name '_110'.

```

Comment:
Should we consider just parsing out the rest of the number rather than just stop? Then we'd avoid the excess `Cannot find name` errors.

Reply:
The proposed grammar says the production is effectively `Digit Sep Digit`, so we'd be much more lenient than is needed. Which seems fine; I don't think `10__01` can be treated as anything other than a number with too many separators.